src/examples/poi-crosschecker/poi-crosschecker.ts

```diff
@@ -4,23 +4,46 @@ import { Observer } from "../../observer";
 import { Messenger } from "../../messenger";
 import { ClientManager } from "../../ethClient";
 import { fetchAllocations, fetchPOI, updateCostModel } from "./queries";
-import {
-  Attestation,
-  defaultModel,
-  printNPOIs,
-  processAttestations,
-  storeAttestations,
-} from "./utils";
+import { defaultModel, processAttestations } from "./utils";
+
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const sqlite3 = require("sqlite3").verbose();
+
+// TODO: Extract to types file
+export type NPOIRecord = {
+  subgraph: string;
+  block: number;
+  nPOI: string;
+  operator: string;
```

Comment:
since `gossipOperatorOf` is an array and indexer `setOperator` doesn't check if the input operator already has an indexer (1:n relationship), should we also specify the indexer?
Maybe we should also have `indexerAddress` or `graphAccount` as an user input instead of just using 0-index for getOperators

Reply:
Totally makes sense, I don't think we should have the indexer address as a user input param but we can definitely keep track of it in the attestation record